contributors/design-proposals/scheduling/nodeaffinity.md

```diff
@@ -144,12 +144,36 @@ Hopefully this won't cause too much confusion.
 
 ## Examples
 
-**TODO: fill in this section**
-
-* Run this pod on a node with an Intel or AMD CPU
-
-* Run this pod on a node in availability zone Z
-
+Run a pod on a node with an Intel or AMD CPU and in availability zone Z:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-with-node-affinity
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: kubernetes.io/arch
+            operator: In
+            values:
+            - intel
+            - amd64
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+          - key: failure-domain.beta.kubernetes.io/zone
```

Comment:
The label is already promoted to GA. Please use `failure-domain.kubernetes.io/zone`

Reply:
Oh, I'm sorry. I'll update it and squash commits today.